clang/test/CodeGen/builtins-image-load-2d-f32.c

```diff
@@ -0,0 +1,31 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
+// RUN: %clang_cc1 -triple amdgcn-unknown-unknown %s -emit-llvm -o - | FileCheck %s
+
+#pragma OPENCL EXTENSION cl_khr_fp64:enable
+
+typedef int v8i __attribute__((ext_vector_type(8)));
```

Comment:
Let's not continue with this ugly typedef in other tests, use int8. Or better, if we had a proper opaque type for images